Helper/Nexus.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Taxjar_SalesTax
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @copyright  Copyright (c) 2017 TaxJar. TaxJar is a trademark of TPS Unlimited, Inc.
+ * @license    http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+
+namespace Taxjar\SalesTax\Helper;
+
+class Nexus extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    private \Taxjar\SalesTax\Model\Tax\NexusFactory $nexusFactory;
+
+    /**
+     * Nexus constructor.
+     *
+     * @param \Magento\Framework\App\Helper\Context $context
+     * @param \Taxjar\SalesTax\Model\Tax\NexusFactory $nexusFactory
+     */
+    public function __construct(
+        \Magento\Framework\App\Helper\Context $context,
+        \Taxjar\SalesTax\Model\Tax\NexusFactory $nexusFactory
+    ) {
+        parent::__construct($context);
+        $this->nexusFactory = $nexusFactory;
+    }
+
+    /**
+     * Get nexus collection formatted as an array of address data arrays
+     *
+     * @param string|int|null $storeId optionally filter result by store_id
+     * @return array
+     */
+    public function getNexusAddresses($storeId): array
+    {
+        /** @var array|\Taxjar\SalesTax\Api\Data\Tax\NexusInterface[] $nexusArray */
+        $nexusArray = $this->getNexusCollection($storeId)->getItems();
+        return array_map(fn ($nexus) => $this->getNexusData($nexus), $nexusArray);
```

Comment:
When making a request to v2/taxes I am getting a 406 response with this body:

{"status":"406","error":"Not Acceptable","detail":"nexus_addresses is invalid"}

The nexus_addresses parameter for that request contains this data:

{"42":{"id":"42","country":"US","zip":"84651","state":"UT","city":"Payson","street":"123 Main St"}}

I believe that having it start at index 42 may be the problem (haven't tested that though) and that it comes from the output of this method.

Reply:
Fixed and updated unit tests to account for this